docs/content/concepts/resources.mdx

```diff
@@ -170,9 +180,9 @@ For more information on using environment variables with Dagster, refer to the [
 
 ### Configuring resources at launch time
 
-In some cases, you may want to specify configuration for a resource at launch time, in the launchpad or in a <PyObject object="RunRequest" /> for a [schedule](/concepts/partitions-schedules-sensors/schedules) or [sensor](/concepts/partitions-schedules-sensors/sensors). For example, you may want a sensor-triggered run to specify a different target table in a database resource for each run.
+In some cases, you may want to specify configuration for a resource at launch time, in the Launchpad or in a <PyObject object="RunRequest" /> for a [schedule](/concepts/partitions-schedules-sensors/schedules) or [sensor](/concepts/partitions-schedules-sensors/sensors). For example, you may want a sensor-triggered run to specify a different target table in a database resource for each run.
```

Comment:
In line 208 "launchpad" isn't capitalized - can you also capitalize that one? GH won't let me comment on that line

Reply:
Yes, good catch!